addons/mass_mailing/tests/test_mailing_mailing_schedule_date.py

```diff
@@ -31,3 +32,65 @@ def test_mailing_schedule_date(self):
         self.assertEqual(mailing.schedule_date, datetime(2021, 4, 30, 9, 0))
         self.assertEqual(mailing.schedule_type, 'scheduled')
         self.assertEqual(mailing.state, 'in_queue')
+
+    def test_mailing_next_departure(self):
```

Comment:
Linting: could you order test methods alphabetically (aka defining it before test_mailing_schedule_date) ? Reason is that as they are executed alphabetically in a given class, better follow the same order in definition so that execution matches file order.

Reply:
Was not aware of this. Will push updated code once I have implemented any other code changes from the review comments!